internal/gql/client.go

```diff
@@ -0,0 +1,17 @@
+package gql
+
+import (
+	"context"
+
+	"github.com/aviator-co/av/internal/config"
+	"github.com/shurcooL/graphql"
+	"golang.org/x/oauth2"
+)
+
+func GraphQLClient() *graphql.Client {
```

Comment:
In Go, we always use the package name to qualify the names within that package, so the `GraphQL` in this function name is redundant with the `gql` of the package name.

```suggestion
func NewClient() *graphql.Client {
```
might be a more idiomatic name.